extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -232,51 +228,51 @@ $color-rail-background: mix($color-page, $black, 90%);
 
                 .carousel-container {
                     left: 16px;
-                    width: 260px;
+                    width: 220px;
                 }
 
                 .since, .chatter {
-                    width: 32px;
+                    width: 30px;
                 }
+            }
+        }
+    }
+}
 
-                .UserStatsMenu {
-                    outline: rgba($color-divider, .3) solid 2px;
-                    width: 220px;
+.UserStatsMenu .new-chat {
```

Comment:
this will not be useful anymore when we decide to release this stuff together with chat module. This is for now class that turns the chat styling on ( when wikia-rail class is present as well )